### ISSUE TITLE
Chapter 10: callout numbers for annotation texts are not correct in source code block ch09l005 

### ISSUE THREAD

**Issue by paatre** [hjwp/Book-TDD-Web-Dev-Python#226]
`asciidoctor` will warn of the following errors when `make chapter_10_production_readiness.html` is ran:

```sh
asciidoctor: WARNING: chapter_10_production_readiness.asciidoc: line 374: callout list item index: expected 1, got 3
asciidoctor: WARNING: chapter_10_production_readiness.asciidoc: line 376: callout list item index: expected 2, got 4
```

When a source block is annotated by callout numbers, same numbers should be used in the annotation text section. But this is not the case in `chapter_10_production_readiness.asciidoc`:

```asciidoc
.Dockerfile (ch09l005)
====
[source,dockerfile]
----
FROM python:slim

RUN python -m venv /venv
ENV PATH="/venv/bin:$PATH"

COPY requirements.txt requirements.txt  <1>
RUN pip install -r requirements.txt  <2>

COPY src /src

WORKDIR /src

CMD python manage.py runserver
----
====

<3> We COPY our requirements file in, just like the src folder.

<4> Now instead of just installing Django, we install all our dependencies
  by pointing pip at the _requirements.txt_ using the `-r` flag.
  Notice the `-r`.
```

### Changelist

- [ ] Change `<3> We COPY our requirements file in...` into `<1> We COPY our requirements file in...`
- [ ] Change `<4> Now instead of just installing Django...` into `<2> Now instead of just installing Django...`

**Comment by paatre** [comment]
These have been merged in #227.

**Comment by hjwp** [comment]
thanks Teemu!
